impress-org/givewp
# Fix: MoneyPHP support for Bitcoin currencies

Resolves #6744

## Description

This adds a custom aggregated list to include Bitcoin (BTC) in our currency facade. These changes are in relation to MoneyPHP and its ability to find "BTC" within its own list.

## Affects
Currencies

## Visuals

https://user-images.githubusercontent.com/75056371/232568056-d833283f-fc6f-4d7b-b6b3-7f6531e5a445.mov



## Testing Instructions

- Donations->Settings->Currency, Set currency to Bitcoin.
- Visit the Donation & the All Forms page.
- Verify the pages load without errors.

## Pre-review Checklist






---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203760890899570
  - https://app.asana.com/0/0/1204615971928969
